instrumentation/opentelemetry-api-0.4/src/main/java/io/opentelemetry/auto/instrumentation/opentelemetryapi/metrics/UnshadedBatchRecorder.java

```diff
@@ -71,6 +75,16 @@ public BatchRecorder put(final DoubleCounter counter, final double value) {
     return this;
   }
 
+  @Override
+  public BatchRecorder put(LongUpDownCounter longUpDownCounter, long l) {
+    return null;
```

Comment:
just don't want to forget (and in case someone looks at this code before we address)
```suggestion
    // TODO https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/463
    return null;
```